commit 370c8cbf951e62cf482f4e4f7005a59a79445af3 (refs/changes/03/303/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2016-08-16T14:36:44-07:00 (3 years, 2 months ago)
    
    joyent/node-triton-tags#3 "make test" broken after joyent/node-triton-tags#1

diff --git a/test/unit.test.js b/test/unit.test.js
index 8e34b42..e76026f 100644
--- a/test/unit.test.js
+++ b/test/unit.test.js
@@ -155,7 +155,7 @@ var cases = [
     {
         key: 'triton.cns.services',
         str: 'foobar:123123123123',
-        err: /must be within the range 0 - 65535/
+        err: /must be within the range 1 - 65535/
     },
     {
         key: 'triton.cns.services',
